File: rust/scriptedmain.rs
//! Modulino

#![crate_id(name = "scriptedmain")]

extern crate std;

// A dummy function to import into test.rs
pub fn meaning_of_life() -> int {
  //! The answer to life, the universe, and everything.
	return 42;
}

#[allow(dead_code)]
fn main() {
	println!("Main: The meaning of life is {}", meaning_of_life().to_str());
}

File: java/ScriptedMain.java
/** Main script */
public final class ScriptedMain {
  /** utility class */
  private ScriptedMain() {}

  /**
     <p>H2G2 meaning of life</p>
     @return meaning of life
  */
  public static int meaningOfLife() {
    return 42;
  }

  /**
     <p>Main app</p>
     @param args CLI arguments
  */
  public static void main(final String[] args) {
    System.out.println("Main: The meaning of life is " + meaningOfLife());
  }
}
